tests/suites/deploy/deploy_bundles.sh

```diff
@@ -345,13 +328,12 @@ test_deploy_bundles() {
 
 		cd .. || exit
 
-		run "run_deploy_bundle"
+		run "run_deploy_charmhub_bundle"
```

Comment:
todo: rename to `run_deploy_bundle`, given that charm store isn't really a thing any longer, no need to be differentiate by saying `charmhub`.